My Nintendo/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "ACertainCoder",
+    "id": "320507572455079949"
+  },
+  "service": "My Nintendo",
+  "description": {
+    "en": "Thanks to My Nintendo makes playing software and interacting with Nintendo even more fun. You can earn points and get rewards such as discounts and additional content for games.",
+    "de": "Dank My Nintendo macht das Spielen von Software und die Interaktion mit Nintendo noch mehr Spaß. Du kannst Punkte sammeln und Belohnungen wie Rabatte und Zusatzinhalte für Spiele erhalten."
+  },
+  "url": [
+      "my.nintendo.com"
+  ],
+  "version": "1.0",
+  "logo": "https://i.imgur.com/eywFbXQ.png",
+  "thumbnail": "https://i.imgur.com/ekWK1iw.png",
+  "color": "#e60012",
+  "tags": ["my", "nintendo", "switch", "ds", "2ds", "3ds"],

```

Comment:
Remove the "my" tag

Reply:
Done 👍

Reply:
if you need to add two words in a tag next time just put a dash between them : "my-nintendo"

Reply:
Ok, thanks